mockwebserver/src/main/kotlin/mockwebserver3/Stream.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2022 Block, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package mockwebserver3
+
+import okio.BufferedSink
+import okio.BufferedSource
+
+/**
+ * A bidirectional sequence of data frames exchanged between client and server.
+ */
+interface Stream {
```

Comment:
I'll keep nerd sniping with suggestions this should be a SocketLike api in okio.

Reply:
Agreed. The giant worm can we’ll find ourselves in is TLS. That’s okay.